permissions/plugin-build-monitor-plugin.yml

```diff
@@ -1,6 +1,9 @@
 ---
 name: "build-monitor-plugin"
+github: &gh "jan-molak/jenkins-build-monitor-plugin"
```

Comment:
github issues are only supported for plugins hosted in jenkinsci.

Which is most likely why the build is failing.